module/purpose/PS3Walk/src/PS3Walk.hpp

```diff
@@ -17,61 +17,76 @@
-        static constexpr uint AXIS_LEFT_JOYSTICK_HORIZONTAL  = 0;
         static constexpr uint AXIS_LEFT_JOYSTICK_VERTICAL    = 1;
-        static constexpr uint AXIS_RIGHT_JOYSTICK_HORIZONTAL = 2;
-        static constexpr uint AXIS_RIGHT_JOYSTICK_VERTICAL   = 3;
+        static constexpr uint AXIS_LEFT_JOYSTICK_HORIZONTAL  = 0;
+        static constexpr uint AXIS_RIGHT_JOYSTICK_HORIZONTAL = 3;
+        static constexpr uint AXIS_RIGHT_JOYSTICK_VERTICAL   = 4;
         static constexpr uint AXIS_L2                        = 12;
         static constexpr uint AXIS_R2                        = 13;
         static constexpr uint AXIS_ACCEL_Y                   = 23;
         static constexpr uint AXIS_ACCEL_X                   = 24;
         static constexpr uint AXIS_ACCEL_Z                   = 25;
 
         // buttons
-        static constexpr uint BUTTON_SELECT         = 0;
-        static constexpr uint BUTTON_LEFT_JOYSTICK  = 1;
-        static constexpr uint BUTTON_RIGHT_JOYSTICK = 2;
-        static constexpr uint BUTTON_START          = 3;
-        static constexpr uint BUTTON_DPAD_UP        = 4;
-        static constexpr uint BUTTON_DPAD_RIGHT     = 5;
-        static constexpr uint BUTTON_DPAD_DOWN      = 6;
-        static constexpr uint BUTTON_DPAD_LEFT      = 7;
-        static constexpr uint BUTTON_L2             = 8;
-        static constexpr uint BUTTON_R2             = 9;
-        static constexpr uint BUTTON_L1             = 10;
-        static constexpr uint BUTTON_R1             = 11;
-        static constexpr uint BUTTON_TRIANGLE       = 12;
-        static constexpr uint BUTTON_CIRCLE         = 13;
-        static constexpr uint BUTTON_CROSS          = 14;
-        static constexpr uint BUTTON_SQUARE         = 15;
+        static constexpr uint BUTTON_SELECT         = 8;
+        static constexpr uint BUTTON_LEFT_JOYSTICK  = 11;
+        static constexpr uint BUTTON_RIGHT_JOYSTICK = 12;
+        static constexpr uint BUTTON_START          = 9;
+        static constexpr uint BUTTON_DPAD_UP        = 13;
+        static constexpr uint BUTTON_DPAD_RIGHT     = 16;
+        static constexpr uint BUTTON_DPAD_DOWN      = 14;
+        static constexpr uint BUTTON_DPAD_LEFT      = 15;
+        static constexpr uint BUTTON_L2             = 6;
+        static constexpr uint BUTTON_R2             = 7;
+        static constexpr uint BUTTON_L1             = 4;
+        static constexpr uint BUTTON_R1             = 5;
+        static constexpr uint BUTTON_TRIANGLE       = 2;
+        static constexpr uint BUTTON_CIRCLE         = 1;
+        static constexpr uint BUTTON_CROSS          = 0;
+        static constexpr uint BUTTON_SQUARE         = 3;
+        static constexpr uint BUTTON_HOME           = 10;
 
         /// @brief Called by the powerplant to build and setup the PS3Walk reactor.
         explicit PS3Walk(std::unique_ptr<NUClear::Environment> environment);
 
     private:
+        /// @brief Stores configuration values
```

Comment:
When you comment things, make sure your comment adds value. Describe how it's used, saying that this stores configuration values doesn't add much to a struct called config :P

Reply:
This is from `./b module generate` script

Reply:
This was a new module when the branch started. But back when I removed non-Director modules, I added in this module myself with Director conversions. So all the additions/removals aren't really building off the existing module in main, they're just differences with the module I made and this version.